scripts/format.sh

```diff
@@ -22,3 +21,15 @@ echo "Using clang-format $CLANG_FORMAT_VERSION from ${CLANG_FORMAT}"
 
 find src valhalla test bench -type f -name '*.h' -o -name '*.cc' \
   | xargs -I{} -P ${NPROC} ${CLANG_FORMAT} -i -style=file {}
+
+
+# Python setup
+py=$(setup_python)
+${py} -m pip install black==22.10.0 flake8==5.0.4
+python_sources=$(LANG=C find scripts src/bindings/python -type f -exec file {} \; | grep -F "Python script" | sed 's/:.*//')
```

Comment:
uses `type` to figure out if a file is a python script or not. we dont always use the file extensions because some of the scripts are part of the installation package